src/screens/Customers/CustomerMain.tsx

```diff
@@ -201,14 +205,14 @@ function CustomerMain(props: CustomerMainProps) {
         onChange={changeTab}
         variant="scrollable"
       >
-        <Tab className={classes.tab} label="All" value={CustomerStatus.ALL} />
+        <Tab className={classes.tab} label={intl(words.all)} value={CustomerStatus.ALL} />
         <Tab className={classes.tab} label={getMeterTabLabel()} value={CustomerStatus.METER} />
         <Tab className={classes.tab} label={getPaymentTabLabel()} value={CustomerStatus.PAYMENT} />
-        <Tab className={classes.tab} label="Done" value={CustomerStatus.DONE} />
+        <Tab className={classes.tab} label={intl(words.done)} value={CustomerStatus.DONE} />
       </Tabs>
       <div className={classes.indicatorContainer}>
         <FiberManualRecordIcon className={classes.indicator}/>
-        <Typography>Status: Active</Typography>
+        <Typography>{intl(words.active_user)}</Typography>
```

Comment:
I think this is wrong, should not say Active [User]

![image](https://user-images.githubusercontent.com/35501399/116501882-d9415480-a866-11eb-9d88-fa260ed7bd96.png)